WCON_format.md

```diff
@@ -295,7 +296,7 @@ An example WCON file with a complete metadata section is given below.
      },
     "units":{"t":"s", "x":"mm", "y":"mm", "temperature":"C", "humidity":"%", "size":"mm", "age":"h"},
     "data":[
-        { "id":1, "t":1.3, "x":-5.3, "y":6.4, "@suzq":[true, true, false, true] }
+        { "id":1, "t":1.3, "x":[-5.3], "y":[6.4], "@suzq":[true, true, false, true] }
```

Comment:
It's harder to read these with the brackets.  And this is meant to be a readable example.  Are they necessary?